staging/src/k8s.io/apiserver/pkg/server/genericapiserver_graceful_termination_test.go

```diff
@@ -200,10 +202,15 @@ func TestGracefulTerminationWithKeepListeningDuringGracefulTerminationDisabled(t
 	}, nil)
 
 	// start the API server
-	stopCh, runCompletedCh := make(chan struct{}), make(chan struct{})
+	_, ctx := ktesting.NewTestContext(t)
+	ctx, cancel := context.WithCancelCause(ctx)
```

Comment:
Will change.

Reply:
I also renamed "cancel" -> "stop" for symmetry.

Reply:
I need to point out that shadowing the `ctx` variable from `NewTestContext` also has its benefit: it makes it impossible to accidentally use `ctx` instead of `stopCtx`.